Sources/Classes/core/RealMiniApp.swift

```diff
@@ -438,6 +438,12 @@ internal class RealMiniApp {
 }
 
 extension RealMiniApp: MiniAppMessageDelegate {
+    #if RMA_SDK_IAP
```

Comment:
@Climbatize @corycaywood We didn't move the `MiniAppPurchaseDelegate` into a separate module/subspec.
Subspec contains only the Host app implementation, so I guess this check is not required.
Do you think we should move the MiniAppPurchaseDelegate as well into separate subspec ?

Reply:
@rleojoseph I'm fine with you removing this check, as long as the Sample App compiles when you remove the IAP dependency from the podfile 👍

Reply:
@rleojoseph I think it's correct to have `MiniAppPurchaseDelegate` in the main module, but like mentioned by @Climbatize it doesn't seem like this check is needed.

On Android, the main purposed of having a separate module is to separate the dependency on the Billing library. But in the case of iOS, it's still good to have a separate module just to separate out the implementation in case the Host App wants to use their own.